source/gui/settingsDialogs.py

```diff
@@ -1974,16 +1970,6 @@ def makeSettings(self, settingsSizer):
 	def postInit(self):
 		self.dictList.SetFocus()
 
-	def onListChar(self, evt):
-		if evt.KeyCode == wx.WXK_RETURN:
-			# The enter key should be propagated to the dialog and thus activate the default button,
-			# but this is broken (wx ticket #3725).

```

Comment:
Perhaps you can add the details of this comment to the `_enterActivatesOk_ctrlSActivatesApply`. It would be easy for the bug in #6206 to return.